examples/HelloWorld/HelloWorld.ps1

```diff
@@ -0,0 +1,35 @@
+try {
+    # Get the path of the script being executed
+    $ScriptPath = (Split-Path -Parent -Path (Split-Path -Parent -Path $MyInvocation.MyCommand.Path))
+    # Get the parent directory of the script's path
+    $podePath = Split-Path -Parent -Path $ScriptPath
+
+    # Check if the Pode module file exists in the specified path
+    if (Test-Path -Path "$($podePath)/src/Pode.psm1" -PathType Leaf) {
+        # If the Pode module file exists, import it
+        Import-Module "$($podePath)/src/Pode.psm1" -Force -ErrorAction Stop
+    }
+    else {
+        # If the Pode module file does not exist, import the Pode module from the system
+        Import-Module -Name 'Pode' -MaximumVersion 2.99 -ErrorAction Stop
+    }
+}
+catch {
+    # If there is any error during the module import, throw the error
+    throw
+}
+
+# Alternatively, you can directly import the Pode module from the system
+# Import-Module Pode
+
+# Start the Pode server
+Start-PodeServer -Browse {
```

Comment:
It might be worth removing `-Browse`, as for me this always caused the server to crash - since the port is forwarded.

I even tried with a clever combination like the following, but it still always failed:
```powershell
https://$($env:CODESPACE_NAME)-<PORT>.$($env:GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN)/
```

Reply:
I just tried a new Codespace and I was able to do everything 
```
PowerShell Extension v2024.2.2
Copyright (c) Microsoft Corporation.

https://aka.ms/vscode-powershell
Type 'help' to get help.

PS /workspaces/Pode> invoke-build test
Build test /workspaces/Pode/pode.build.ps1
Task /test/Build/BuildDeps
Done /test/Build/BuildDeps 00:00:00.7058118
Task /test/Build
SDK for target framework netstandard2.0 is compatible with the installed SDKs
Assembly Version 0.0.0
MSBuild version 17.8.5+b5265ef37 for .NET
  Determining projects to restore...
  Restored /workspaces/Pode/src/Listener/Pode.csproj (in 5.66 sec).
  Pode -> /workspaces/Pode/src/Listener/bin/Release/netstandard2.0/Pode.dll
  Pode -> /workspaces/Pode/src/Libs/netstandard2.0/
SDK for target framework net6.0 is compatible with the installed SDKs
Assembly Version 0.0.0
MSBuild version 17.8.5+b5265ef37 for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
  Pode -> /workspaces/Pode/src/Listener/bin/Release/net6.0/Pode.dll
  Pode -> /workspaces/Pode/src/Libs/net6.0/
SDK for target framework net8.0 is compatible with the installed SDKs
Assembly Version 0.0.0
MSBuild version 17.8.5+b5265ef37 for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
  Pode -> /workspaces/Pode/src/Listener/bin/Release/net8.0/Pode.dll
  Pode -> /workspaces/Pode/src/Libs/net8.0/
Done /test/Build 00:00:47.4823847
Task /test/TestNoBuild/TestDeps
Installing Pester v5.5.0
Done /test/TestNoBuild/TestDeps 00:00:05.3042845                                                                        
Task /test/TestNoBuild

Starting discovery in 36 files.
Discovery found 5547 tests in 4.37s.
Running tests.
[+] /workspaces/Pode/tests/unit/_.Tests.ps1 6.54s (6.12s|209ms)
[+] /workspaces/Pode/tests/unit/Authentication.Tests.ps1 1.01s (883ms|98ms)
[+] /workspaces/Pode/tests/unit/Context.Tests.ps1 2.07s (1.8s|203ms)
[+] /workspaces/Pode/tests/unit/Cookies.Tests.ps1 970ms (713ms|193ms)
[+] /workspaces/Pode/tests/unit/CronParser.Tests.ps1 902ms (645ms|131ms)
[+] /workspaces/Pode/tests/unit/Cryptography.Tests.ps1 185ms (111ms|33ms)
[+] /workspaces/Pode/tests/unit/Endware.Tests.ps1 211ms (168ms|32ms)
[+] /workspaces/Pode/tests/unit/Flash.Tests.ps1 276ms (170ms|58ms)
[+] /workspaces/Pode/tests/unit/Handlers.Tests.ps1 160ms (92ms|24ms)
[+] /workspaces/Pode/tests/unit/Headers.Tests.ps1 195ms (102ms|42ms)
[+] /workspaces/Pode/tests/unit/Helpers.Tests.ps1 2.34s (1.01s|639ms)
[+] /workspaces/Pode/tests/unit/Localization.Tests.ps1 8.51s (2.97s|3.86s)
[+] /workspaces/Pode/tests/unit/Logging.Tests.ps1 193ms (138ms|43ms)
[+] /workspaces/Pode/tests/unit/Mappers.Tests.ps1 1.93s (859ms|756ms)
[+] /workspaces/Pode/tests/unit/Metrics.Tests.ps1 117ms (86ms|24ms)
[+] /workspaces/Pode/tests/unit/Middleware.Tests.ps1 1.01s (777ms|145ms)
[+] /workspaces/Pode/tests/unit/NameGenerator.Tests.ps1 105ms (93ms|10ms)
[+] /workspaces/Pode/tests/unit/OpenApi.Tests.ps1 2.53s (1.97s|274ms)
[+] /workspaces/Pode/tests/unit/PrivateOpenApi.Tests.ps1 505ms (312ms|104ms)
[+] /workspaces/Pode/tests/unit/Responses.Tests.ps1 818ms (672ms|105ms)
[+] /workspaces/Pode/tests/unit/Routes.Tests.ps1 1.39s (1.06s|259ms)
[+] /workspaces/Pode/tests/unit/Schedules.Tests.ps1 396ms (316ms|59ms)
[+] /workspaces/Pode/tests/unit/Security.Tests.ps1 643ms (497ms|105ms)
Pode [dev] (PID: 1474)
Pode [dev] (PID: 1474)
Pode [dev] (PID: 1474)
Pode [dev] (PID: 1474)
[+] /workspaces/Pode/tests/unit/Server.Tests.ps1 576ms (548ms|21ms)
[+] /workspaces/Pode/tests/unit/Serverless.Tests.ps1 410ms (373ms|24ms)
[+] /workspaces/Pode/tests/unit/Sessions.Tests.ps1 358ms (205ms|58ms)
[+] /workspaces/Pode/tests/unit/State.Tests.ps1 221ms (159ms|38ms)
[+] /workspaces/Pode/tests/unit/Timers.Tests.ps1 197ms (135ms|37ms)
[+] /workspaces/Pode/tests/integration/Authentication.Tests.ps1 12.94s (12.88s|38ms)
[+] /workspaces/Pode/tests/integration/Endpoints.Tests.ps1 12.37s (12.35s|16ms)
[+] /workspaces/Pode/tests/integration/RestApi.Https.Tests.ps1 15.94s (15.88s|47ms)
[+] /workspaces/Pode/tests/integration/RestApi.Tests.ps1 12.07s (12s|45ms)
[+] /workspaces/Pode/tests/integration/Schedules.Tests.ps1 15.74s (15.65s|47ms)
[+] /workspaces/Pode/tests/integration/Sessions.Tests.ps1 17.95s (17.91s|22ms)
[+] /workspaces/Pode/tests/integration/Timers.Tests.ps1 12.22s (12.2s|11ms)
[+] /workspaces/Pode/tests/integration/WebPages.Tests.ps1 12.03s (12s|18ms)
Tests completed in 146.13s
Tests Passed: 5547, Failed: 0, Skipped: 0 NotRun: 1
Task /test/TestNoBuild/PushCodeCoverage skipped.
Task /test/TestNoBuild/CheckFailedTests
Done /test/TestNoBuild/CheckFailedTests 00:00:00.0018872
Done /test/TestNoBuild 00:02:33.2755427
Done /test 00:03:20.8328561
Build succeeded. 6 tasks, 0 errors, 0 warnings 00:03:22.6289798
PS /workspaces/Pode> 
```

Reply:
The environment is brand new

Reply:
opps wrong thread 
I see I removed -Browse